isobus/include/isobus/isobus/isobus_guidance_interface.hpp

```diff
@@ -191,36 +196,41 @@ namespace isobus
 				NotAvailable = 63 ///< Parameter not supported
 			};
 
-			/// @brief Constructor for a AgriculturalGuidanceMachineInfo
-			AgriculturalGuidanceMachineInfo() = default;
+			/// @brief Constructor for a GuidanceMachineInfo
+			/// @param[in] sender The control function that is sending this message
+			GuidanceMachineInfo(ControlFunction *sender);
```

Comment:
Since these constructors are no longer `default` we should mark them as `explicit`